src/Microsoft.AspNet.Mvc.Abstractions/Filters/ResourceExecutingContext.cs

```diff
@@ -29,12 +29,6 @@ public ResourceExecutingContext(ActionContext actionContext, IList<IFilterMetada
         /// </summary>
         public virtual FormatterCollection<IInputFormatter> InputFormatters { get; set; }
 
-        /// <summary>
-        /// Gets or sets the list of <see cref="IOutputFormatter"/> instances used to format the response.
-        /// </summary>
-        public virtual FormatterCollection<IOutputFormatter> OutputFormatters { get; set; }
```

Comment:
As discussed, removing this from ResourceFilters. All these scenarios still work with result filters.